views/lib/extended_footer.hbs

```diff
@@ -1,30 +1,30 @@
 {{#embed "authentication/section" style="dark footer" class=""}}
 {{#content "body"}}
-<footer id="footer" tabindex="0">
+<footer id="footer">
 	<div class="container">
 		<div class="row">
 			<div class="col-md-4">
-				<h2 class="h4">{{$t "lib.extended_footer.heading.contact" }}</h2>
+				<h2 class="h4">{{$t "global.link.contact" }}</h2>
 				<ul>
 					{{> "lib/extended_footer_links"}}
-					<li><a href="https://hpi.de/open-campus/registrierung/hpi-schul-cloud-newsletter/" target="_blank" rel="noopener" aria-label="{{$t "lib.extended_footer.link.subscribeTheNewsletter" }} {{$t "global.openInNewTabWarning"}}">
+					<li><a href="https://hpi.de/open-campus/registrierung/hpi-schul-cloud-newsletter/" target="_blank" rel="noopener">
```

Comment:
aria-label="{{$t "lib.extended_footer.link.subscribeTheNewsletter" }} {{$t "global.openInNewTabWarning"}} was removed here

Reply:
some more done this file. They are important for a11y

Reply:
done ;)